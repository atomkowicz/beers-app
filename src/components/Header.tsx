import * as React from 'react';
import {Link} from "react-router-dom";
import '../assests/css/header.css';
import logo from '../assests/img/app-logo.png';

const Header = () => {
    return (
        <header className="Header">
            <Link to={'/'}><img src={logo} className="App-logo" alt="logo"/></Link>
        </header>
    );
}

export default Header;