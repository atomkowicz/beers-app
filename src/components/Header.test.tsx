import * as React from 'react';
import {MemoryRouter} from "react-router-dom";
import * as renderer from 'react-test-renderer';
import Header from "./Header";

it('header renders correctly', () => {
    const tree = renderer
        .create(<MemoryRouter><Header/></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

