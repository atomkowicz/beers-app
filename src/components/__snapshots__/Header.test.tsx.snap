// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<header
  className="Header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="logo"
      className="App-logo"
    />
  </a>
</header>
`;
